virtcontainers/sandbox_test.go

```diff
@@ -1722,3 +1722,29 @@ func TestGetNetNs(t *testing.T) {
 	netNs = s.GetNetNs()
 	assert.Equal(t, netNs, expected)
 }
+
+func TestAdjustVCPUCount(t *testing.T) {
```

Comment:
Thanks for adding this. However, there are quite a few missing scenarios for this test (to handle all the `if` tests in `adjustVCPUCount()`).